### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by KaspianDev** [DragonEggBedrockBreaking/VanillaDisable#8]
```
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2/24/22, 5:56 PM
Description: Ticking entity

java.lang.NullPointerException: Cannot invoke "net.minecraft.server.MinecraftServer.method_3767()" because the return value of "uk.debb.vanilla_disable.gamerules.RegisterGamerules.getServer()" is null
	at net.minecraft.class_1297.handler$eco000$cannotBeInvisible(class_1297.java:22036)
	at net.minecraft.class_1297.method_5767(class_1297.java)
	at net.minecraft.class_1297.method_5756(class_1297.java:2233)
	at net.minecraft.class_1309.handler$zdk001$tick(class_1309.java:5090)
	at net.minecraft.class_1309.method_5773(class_1309.java)
	at net.minecraft.class_1657.method_5773(class_1657.java:274)
	at net.minecraft.class_742.method_5773(class_742.java:624)
	at net.minecraft.class_746.method_5773(class_746.java:215)
	at net.minecraft.class_638.method_18646(class_638.java:218)
	at net.minecraft.class_1937.method_18472(class_1937.java:487)
	at net.minecraft.class_638.method_32124(class_638.java:201)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:197)
	at net.minecraft.class_310.method_1574(class_310.java:1751)
	at net.minecraft.class_310.method_1523(class_310.java:1086)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_1297.handler$eco000$cannotBeInvisible(class_1297.java:22036)
	at net.minecraft.class_1297.method_5767(class_1297.java)
	at net.minecraft.class_1297.method_5756(class_1297.java:2233)
	at net.minecraft.class_1309.handler$zdk001$tick(class_1309.java:5090)
	at net.minecraft.class_1309.method_5773(class_1309.java)
	at net.minecraft.class_1657.method_5773(class_1657.java:274)
	at net.minecraft.class_742.method_5773(class_742.java:624)
	at net.minecraft.class_746.method_5773(class_746.java:215)
	at net.minecraft.class_638.method_18646(class_638.java:218)
	at net.minecraft.class_1937.method_18472(class_1937.java:487)
	at net.minecraft.class_638.method_32124(class_638.java:201)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:197)

-- Entity being ticked --
Details:
	Entity Type: minecraft:player (net.minecraft.class_746)
	Entity ID: 105
	Entity Name: KapceRBLX
	Entity's Exact location: 8.50, 65.00, 8.50
	Entity's Block location: World: (8,65,8), Section: (at 8,1,8 in 0,4,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at net.minecraft.class_1937.method_18472(class_1937.java:487)
	at net.minecraft.class_638.method_32124(class_638.java:201)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:197)
	at net.minecraft.class_310.method_1574(class_310.java:1751)
	at net.minecraft.class_310.method_1523(class_310.java:1086)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Affected level --
Details:
	All players: 1 total; [class_746['KapceRBLX'/105, l='ClientLevel', x=8.50, y=65.00, z=8.50]]
	Chunk stats: 1024, 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Section: (at 8,0,8 in 0,4,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 0 game time, 0 day time
	Server brand: ~~NULL~~
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:408)
	at net.minecraft.class_310.method_1587(class_310.java:2402)
	at net.minecraft.class_310.method_1514(class_310.java:752)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, Basic Nether Ores 16x.zip, Copper Legacy Pack.zip, artifality/fancyclusters

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 509221888 bytes (485 MiB) / 1782579200 bytes (1700 MiB) up to 1782579200 bytes (1700 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz
	Identifier: Intel64 Family 6 Model 60 Stepping 3
	Microarchitecture: Haswell (Client)
	Frequency (GHz): 3.20
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce GT 730
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 2048.00
	Graphics card #0 deviceId: 0x1287
	Graphics card #0 versionInfo: DriverVersion=27.21.14.6589
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 1.60
	Memory slot #0 type: DDR3
	Virtual memory max (MB): 17352.02
	Virtual memory used (MB): 11528.14
	Swap memory total (MB): 9216.00
	Swap memory used (MB): 531.34
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx1700m
	Fabric Mods: 
		additionaladditions: Additional Additions 3.0.1
		advancednetherite: Advanced Netherite 1.4.0-1.18.1
		alternatecurrent: Alternate Current 1.1.0
		amecsapi: Amecs API 1.3.3+mc1.18-pre1
		apoli: Apoli 2.2.2
		appleskin: AppleSkin 2.3.0+mc1.18.1
		architectury: Architectury 3.7.20
		artifacts: Artifacts 5.0.2+fabric
		artifality: Artifality 0.5.5
		attributefix: AttributeFix 13.0.5
		backslot: BackSlot 1.2.5
		bendy-lib: Bendy api 2.0.4
		blue_endless_jankson: jankson 1.2.1
		bno: Basic Nether Ores 1.18.1-7.1.2-Fabric
		byg: BYG 1.3.5.10
		caffeineconfig: CaffeineConfig 1.0.0
		calio: Calio 1.4.2
		cardinal-components-base: Cardinal Components API (base) 4.0.0
		cardinal-components-entity: Cardinal Components API (entities) 4.0.0
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-config: Cloth Config v6 6.2.57
		clumps: Clumps 8.0.0+5
		com_electronwill_night-config_core: core 3.6.4
		com_electronwill_night-config_toml: toml 3.6.4
		com_github_wearblackallday_javautils: JavaUtils bd58640372
		com_velocitypowered_velocity-native: velocity-native 1.1.0-SNAPSHOT
		cosmetic-armor: Cosmetic Armor 1.2.1
		crowdin-translate: CrowdinTranslate 1.4+1.18
		customportalapi: Custom Portal Api 0.0.1-beta49-1.18
		damagetilt: Damage Tilt 1.18-fabric-0.1.2
		decorative_blocks: Decorative Blocks 2.0.4
		defaultsettings: DefaultSettings 3.1.0
		dimthread: DimensionalThreading 1.2.6
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.4
		drinkbeer: Drink Beer 3.0.2
		emotecraft: Emotecraft 2.1
		expandability: ExpandAbility 5.0.0
		extraorigins: Extra Origins 1.18-9
		fabric: Fabric API 0.46.4+1.18
		fabric-api-base: Fabric API Base 0.4.2+d7c144a865
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.3+d7c144a865
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.2+d7c144a865
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.10+3ac43d9565
		fabric-command-api-v1: Fabric Command API (v1) 1.1.7+d7c144a865
		fabric-commands-v0: Fabric Commands (v0) 0.2.6+b4f4f6cd65
		fabric-containers-v0: Fabric Containers (v0) 0.1.19+d7c144a865
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.8+d7c144a865
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.9+3ac43d9565
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.10+a1d9bbf565
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.6+d7c144a865
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.17+d7c144a865
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.9+d7c144a865
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.11+d7c144a865
		fabric-item-api-v1: Fabric Item API (v1) 1.3.1+691a79b565
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.6+3ac43d9565
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.9+d7c144a865
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.7+b4f4f6cd65
		fabric-language-scala: Fabric Language Scala 1.1.0+scala.2.13.6
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.13+713c266865
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.9+d7c144a865
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.6+d7c144a865
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.11+b4f4f6cd65
		fabric-models-v0: Fabric Models (v0) 0.3.4+d7c144a865
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.19+d7c144a865
		fabric-networking-v0: Fabric Networking (v0) 0.3.6+b4f4f6cd65
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.4+d7c144a865
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.12+d7c144a865
		fabric-particles-v1: Fabric Particles (v1) 0.2.10+526dc1ac65
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.2+ad01bfbd65
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.11+b0b66fc365
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.15+6825030165
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.10+b4f4f6cd65
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.5+d7c144a865
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.19+3ac43d9565
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.12+b4f4f6cd65
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.6+713c266865
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 1.0.2+d7c144a865
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.14+713c266865
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.8+d7c144a865
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.12+d7c144a865
		fabric-structure-api-v1: Fabric Structure API (v1) 2.1.3+d7c144a865
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.8+d7c144a865
		fabric-textures-v0: Fabric Textures (v0) 1.0.10+3ac43d9565
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.8+fb3b57b465
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.9+d7c144a865
		fabricloader: Fabric Loader 0.13.2
		fabricshieldlib: Fabric Shield Lib 1.5.1-pre1-1.18
		feature_nbt_deadlock_be_gone: Feature NBT Deadlock Be Gone - Fabric 1.0.1+1.18.1
		ferritecore: FerriteCore 4.1.2
		friendsandfoes: Friends&Foes 1.2.4
		genesis: Genesis 1.18.1-1.0.0
		goblintraders: Goblin Traders 1.5
		guardvillagers: Guard Villagers Fabric 1.0.7
		hologram-api: Hologram API 0.2.1+1.18-pre5
		impactfulweather: Biome Particle Weather 1.3.1
		indium: Indium 1.0.2-alpha2+mc1.18.1
		inventory_backpack: Inventory Backpack 0.2.4
		jankson: Jankson 4.0.0+j1.2.0
		java: OpenJDK 64-Bit Server VM 17
		javax_servlet_javax_servlet-api: javax.servlet-api 3.1.0
		kanos_config: Kanos Config 0.1.4+1.14.4-1.17.1
		krypton: Krypton 0.1.6
		ksyxis: Ksyxis 1.1
		levelz: LevelZ 1.1.6
		libgui: LibGui 5.3.0+1.18
		libninepatch: LibNinePatch 1.1.0
		libzoomer: LibZoomer 0.3.0+1.17.1
		lithium: Lithium 0.7.8
		medievalweapons: Medieval Weapons 1.1.8
		minecraft: Minecraft 1.18.1
		minecraftmappings: Minecraft Mappings 1.18-1.0.0
		mm: Manningham Mills 2.3
		modmenu: Mod Menu 3.0.1
		morevillagers-fabric: MoreVillagersFabric 3.1.1-SNAPSHOT
		moreweaponry: More Weaponry 1.15.43+1.18
		mostructures: Mo' Structures 1.3.0-pre1+1.18
		mtr: Minecraft Transit Railway 1.18.1-3.0.0-beta-8
		nbtcrafting: NBT Crafting 2.1.2+mc1.18.1
		no_fog: No Fog 1.2.0+1.16.5-1.18.1
		no_null_processors: No Null Processors - Fabric 2.0.1+1.18.1
		notenoughanimations: NotEnoughAnimations 1.4.0
		okzoomer: Ok Zoomer 5.0.0-beta.3+1.17.1
		omega-config: OmegaConfig 1.2.2-1.18.1
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_eclipse_jetty_jetty-http: jetty-http 9.4.44.v20210927
		org_eclipse_jetty_jetty-io: jetty-io 9.4.44.v20210927
		org_eclipse_jetty_jetty-security: jetty-security 9.4.44.v20210927
		org_eclipse_jetty_jetty-server: jetty-server 9.4.44.v20210927
		org_eclipse_jetty_jetty-servlet: jetty-servlet 9.4.44.v20210927
		org_eclipse_jetty_jetty-util: jetty-util 9.4.44.v20210927
		org_eclipse_jetty_jetty-util-ajax: jetty-util-ajax 9.4.44.v20210927
		org_joml_joml: joml 1.10.2
		org_quiltmc_quilt-json5: quilt-json5 1.0.0
		origins: Origins 1.3.1
		origins-classes: Origins: Classes 1.2.4
		packet_tweaker: Packet Tweaker 0.2.0+1.18-pre1
		pehkui: Pehkui 3.1.0+1.14.4-1.18.1
		placeholder-api: Placeholder API 1.1.3+1.17.1
		playerabilitylib: Pal 1.3.0
		polymer: Polymer (Core) 0.2.0-beta.24+1.18.1
		polymer-legacy: Polymer (Compatibility) 0.1.9+0.2.0-beta.24+1.18.1
		quarryplus: QuarryPlus 18.2.493
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		recipecache: recipecache 0.2.1-1.17.1
		roughlyenoughitems: Roughly Enough Items 7.3.432
		roughlyenoughresources: Roughly Enough Resources 2.4.1
		server_translations_api: Server Translations API 1.4.8+1.18-pre1
		sgui: sgui 1.0.0-rc6+1.18-pre5
		sodium: Sodium 0.4.0-alpha6+build.14
		spark: spark 1.8.8
		spruceui: SpruceUI 3.3.0+1.17
		starlight: Starlight 1.0.0+fabric.d0a3220
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		terrablender: TerraBlender 1.1.0.77
		thonkutil-base: ThonkUtil Base 1.5.1+31ad4eb1c8
		thonkutil-capes-v1: ThonkUtil Capes (v1) 1.2.2+befce073c8
		thonkutil-potions-v0: ThonkUtil Potions (v0) 1.3.1+befce073c8
		thonkutil-trades-v1: ThonkUtil Trades (v1) 1.1.8+d77526a2c8
		trinkets: Trinkets 3.2.0
		twigs: Twigs 1.1.2
		universal-graves: Universal Graves 2.0.0-rc.4+1.18.1
		vanilla_disable: Vanilla Disable 1.3.1
		villager-hats: Villager Hats Mod 1.3.0
		vmp: Very Many Players 0.2.0+beta.3.8
		waystones: Waystones 2.4.1
		wthit: wthit 4.5.2
		xaerominimap: Xaero's Minimap 22.2.0.1
		xaeroworldmap: Xaero's World Map 1.20.3.1
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: NVIDIA GeForce GT 730/PCIe/SSE2 GL version 3.2.0 NVIDIA 465.89, NVIDIA Corporation
	Window size: 854x480
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: vanilla, Fabric Mods, file/Basic Nether Ores 16x.zip, file/Copper Legacy Pack.zip, artifality/fancyclusters (incompatible)
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-4460 CPU @ 3.20GHz
```

only crashes client

**Comment by DragonEggBedrockBreaking** [comment]
It seems here you are playing on a dedicated server. Have you tested this in singleplayer? I suspect that the code is trying to run even when on the client. If this is the case, the mod should work in singleplayer. The fix may not be too difficult.

**Comment by DragonEggBedrockBreaking** [comment]
Fixed in 1.3.2
